From 6e3d38cbab91dba21e6a79b7992ee93c5ba90110 Mon Sep 17 00:00:00 2001
From: make shi <b15407@freescale.com>
Date: Mon, 19 Mar 2012 17:41:14 +0800
Subject: ENGR00177154 solo sabreauto: config USB pin according to board

config the USB pin and power control for ARD board

Signed-off-by: make shi <b15407@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabreauto.c    |    4 ++--
 arch/arm/mach-mx6/board-mx6solo_sabreauto.h |    2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabreauto.c b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
index 2d2eeed..aa144b9 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabreauto.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
@@ -128,7 +128,7 @@
 #define SABREAUTO_CAN_STBY		SABREAUTO_IO_EXP_GPIO2(5)
 #define SABREAUTO_CAN_EN		SABREAUTO_IO_EXP_GPIO2(6)
 #define SABREAUTO_USB_HOST1_PWR		SABREAUTO_IO_EXP_GPIO2(7)
-#define SABREAUTO_USB_OTG_PWR		SABREAUTO_IO_EXP_GPIO3(2)
+#define SABREAUTO_USB_OTG_PWR		SABREAUTO_IO_EXP_GPIO3(1)
 #define BMCR_PDOWN			0x0800 /* PHY Powerdown */
 
 static struct clk *sata_clk;
@@ -680,7 +680,7 @@ static void __init imx6q_sabreauto_init_usb(void)
 	}
 	gpio_direction_input(SABREAUTO_USB_HOST1_OC);
 
-	mxc_iomux_set_gpr_register(1, 13, 1, 1);
+	mxc_iomux_set_gpr_register(1, 13, 1, 0);
 	mx6_set_otghost_vbus_func(imx6q_sabreauto_usbotg_vbus);
 	mx6_usb_dr_init();
 	mx6_set_host1_vbus_func(imx6q_sabreauto_usbhost1_vbus);
diff --git a/arch/arm/mach-mx6/board-mx6solo_sabreauto.h b/arch/arm/mach-mx6/board-mx6solo_sabreauto.h
index 06b1d01..ba29314 100644
--- a/arch/arm/mach-mx6/board-mx6solo_sabreauto.h
+++ b/arch/arm/mach-mx6/board-mx6solo_sabreauto.h
@@ -178,7 +178,7 @@ static iomux_v3_cfg_t mx6dl_sabreauto_pads[] = {
 	MX6DL_PAD_EIM_EB0__GPIO_2_28,
 
 	/* USBOTG ID pin */
-	MX6DL_PAD_ENET_RX_ER__ENET_RX_ER,
+	MX6DL_PAD_ENET_RX_ER__ANATOP_USBOTG_ID,
 
 	/* VIDEO adv7180 INTRQ */
 	MX6DL_PAD_ENET_RXD0__GPIO_1_27,
-- 
1.7.9.5

